src/index.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  background-image: url("../public/images/bgimage.jpg");
+  width: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-size: cover;
+}
+
+.questionz{
+  -webkit-text-stroke: 1px rgba(255,126,25,.45);
+  text-shadow: 1.75px 2px 15px black, 0 0 .5em lightblue, 0 0 0.2em lightblue;
+  color: rgb(247,225,193);
+  font-family: "Book Antiqua",Palatino,"Palatino Linotype","Palatino LT STD",Georgia,serif;
+}
+
+.Header {
+  background-color: rgba(247, 225, 193, .50);
+  box-shadow: inset 0 0 10px #00467A;
+  padding: 10px;
+}
+
+.cardText {
+  font-size: 12px;
+}
+
+.App{
+  box-shadow: inset 0 0 10px #00467A;
+}
+
+input {
```

Comment:
its better to give these inputs unique classnames and or make them their own components so you dont have to style all inputs